src/routes/+page.svelte

```diff
@@ -83,4 +98,41 @@
       {/each}
     </ul>
   {/if}
+  {#if commissionerGreeting?.json && commissionerByline && commissionerHeadshot?.linkedImage?.url}
+    <section class="greeting-wrapper" aria-label="Introduction">
+      <!-- This section can render without the background image, so this is optional. -->
+      {#if commissionerBackground?.linkedImage?.url}
+        <Image
+          class="greeting-background"
+          src={commissionerBackground.linkedImage.url}
+          alt={commissionerBackground.altText ?? ""}
+          blurhash={commissionerBackground.linkedImage.blurhash ?? undefined}
+          height={commissionerBackground.linkedImage.height}
+          width={commissionerBackground.linkedImage.width}
+          preserveAspectRatio={false}
+          canUpscaleImage
+          sources={getSources}
+          loading="lazy"
+        />
```

Comment:
@Benaiah would appreciate your eyes on both this `Image` component as well as the one below. Not sure if I'm handling them exactly right, and it seems like Lighthouse is giving us performance warnings about "properly sizing images": https://meta-site-406-ldaf.vercel.app/lighthouse/mobile/ldaf-hiij13x2i-ldaf_vercel_app_.report.html